nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -129,6 +129,15 @@ def _setup_dataloader_from_config(self, config: Optional[Dict]):
 
         # Instantiate tarred dataset loader or normal dataset loader
         if config.get('is_tarred', False):
+            if ('tarred_audio_filepaths' in config and config['tarred_audio_filepaths'] is None) or (
```

Comment:
Why are you handling this at the model level instead of the dataset level?
Why not add a dataclass that has filepaths as MISSING?
Is this another issue caused by loading pre-trained models?

Reply:
The dataset cannot return an empty instance of itself if it were to handle this input. It's upto the model to not load the dataset if paths are not provided and instead set it as None so that PTL doesn't execute that dataloader. 

And we don't want to crash dataset with missing value since that will break deserilization. Datasets cannot return None themselves inside Init either so it's not the responsibility of the dataset to take the missing case into consideration